android/BUILDING.md

```diff
@@ -0,0 +1,24 @@
+# Build OpenThread Commissioner App
+
+This documents describes instructions to build the OT Commissioner (Android) App.
+
+## Prerequisites
+
+- macOS or Linux computer.
+- Install latest Android Studio from [here](https://developer.android.com/studio).
+
+## Build shared library
+
+The Commissioner Android App is built atop of the native OT Commissioner library. There is a `build-commissioner-libs.sh` script in current directory to help building the required libraries:
+
+```shell
+ANDROID_ABI=arm64-v8a ANDROID_NDK_HOME=$HOME/Library/Android/sdk/ndk/21.3.6528147 ./build-commissioner-libs.sh
+```
+
+This script creates a build directory (`.build-${ANDROID_ABI}`) in current directory and copies generated libraries into target sub-folders in `openthread_commissioner`.
+
+_Note: you need to set env `ANDROID_ABI` to the ABI of your phone. This value can be got with command `adb shell getprop ro.product.cpu.abi` after connecting the phone to your computer._
```

Comment:
Make it a markdown note

> Note: ...text

Exactly how does one "set env `ANDROID_ABI`" ?... you have the command to get the value but I don't know how to set it.

Reply:
The example building script command 

```
ANDROID_ABI=arm64-v8a ANDROID_NDK_HOME=$HOME/Library/Android/sdk/ndk/21.3.6528147 ./build-commissioner-libs.sh
```

shows how we can set the ANDROID_ABI env for the `build-commissioner-libs.sh` script.

Reply:
Ah got it, thanks.